cmd/security-proxy-setup/res/configuration.toml

```diff
@@ -17,7 +17,7 @@
 # This is a TOML config file for edgexsecurity service.
 
 [Writable]
-LogLevel = "ERROR"
+LogLevel = "INFO"
```

Comment:
Why was this changed?

Reply:
Good catch. It came through the merged conflicts with master branch yesterday.  Eventually we want it to be INFO to start with based on another issue by Cloud.

Reply:
changed has been reverted